packages/terra-clinical-application/package.json

```diff
@@ -36,7 +36,7 @@
     "terra-doc-template": "^1.2.0"
   },
   "scripts": {
-    "compile": "npm run compile:clean && npm run compile:build && rm terra-clinical-application.tgz && npm pack --ignore-scripts && mv *.tgz terra-clinical-application.tgz",
+    "compile": "npm run compile:clean && npm run compile:build && tt-pack",
```

Comment:
Just a side node, I am a little worried about naming this script `tt-pack` because we have been telling people to add a `pack` script to projects in referencing to webpack packing assets.

Reply:
I'm writing a note about it on the terra-toolkit PR, we can move discussions over on that when it is opened. 

The challenge is this script is essentially running `npm pack`. https://docs.npmjs.com/cli/pack

Reply:
Yeah... I want our terminology to match npm but we have also made it weird with out script naming preference for webpack. 🤷‍♀️ Just wanted to mention something, mostly so I wouldn't' forget, but lets move discussion there.